.clang-format

```diff
@@ -0,0 +1,428 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
GPL - does this interfere with Zephyr?

Reply:
no, this will not end up in any binary, we have a few of those.